lua/telekasten.lua

```diff
@@ -1446,10 +1456,12 @@ local function RenameNote()
     if M.Cfg.rename_update_links == true then
         -- Only look for the first part of the link, so we do not touch to #heading or #^paragraph
         -- Should use regex instead to ensure it is a proper link
-        -- Should also account for other types of links (?)
```

Comment:
As per the recent discussions, I am confident we'll stick to `[[` for anything 'wiki' or telekasten related, so `[[` alone should be fine!

Reply:
Perfect! This makes my life easier 😄